src/components/ProfilePage/SocialLinks.jsx

```diff
@@ -190,16 +192,15 @@ SocialLink.propTypes = {
   name: PropTypes.string.isRequired,
 };
 
-
 function EditableListItem({
-  url,
-  platform,
-  onClickEmptyContent,
-  name,
+  url, platform, onClickEmptyContent, name,
 }) {
-  const linkDisplay = url != null ?
-    <SocialLink name={name} url={url} platform={platform} /> :
-    <EmptyContent onClick={onClickEmptyContent}>Add {name}</EmptyContent>;
+  const linkDisplay =
+    url != null ? (
```

Comment:
Philosophical note about two things in tension here: the case where it's not null is the one you care more about, so it makes sense to put it first, but on the other hand, conditionals are easiest for humans to read if they're not unpositive.  I tend to prioritize the second consideration more than the first.